language/en_AU/transfer_email_verify_to_download.mail.php

```diff
@@ -0,0 +1,41 @@
+<?php 
+// WARNING, this is a read only file created by import scripts
+// WARNING
+// WARNING,  Changes made to this file will be clobbered
+// WARNING
+// WARNING,  Please make changes on poeditor instead of here
+// 
+// 
+?>
+subject: FileSender: One time verification code for download
```

Comment:
Possibly better to use the cfg:site_name here instead of static string FileSender

Reply:
Cool. I did a few passes to catch if I still had statics in the html/php but hadn't really worked on the email text too much. Updated.